.gitattributes

```diff
@@ -0,0 +1 @@
+/.delivery/ merge=ours
```

Comment:
Can you put this in .git/info/attributes so it doesn't need to be committed with the project? Thanks!